tests/assets/configs/bayes.yaml

```
@@ -0,0 +1,32 @@
+- node_type: embedding
+  target_metric: retrieval_hit_rate
+  search_space:
+    - module_name: retrieval
+      k: [10]
+      embedder_name:
+        - sentence-transformers/all-MiniLM-L6-v2
+        - avsolatorio/GIST-small-Embedding-v0
+- node_type: scoring
+  target_metric: scoring_roc_auc
+  search_space:
+    - module_name: knn
+      k:
+        type: "int"
+        content:
+          low: 5
+          high: 10
+          step: 1
```

Мб как-то так поменять? Кажется тип мы сами можем находить.
```suggestion
      k:
        search_space:
          low: 5
          high: 10
          step: 1
```

---

Если это упростит задачу то ок, да

---

Это упрощает написание конфигов, чтобы не писать лишнее пользователям